devtools/composer.json

```diff
@@ -4,7 +4,7 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "apigen/apigen": "^2.8",
-        "fabpot/php-cs-fixer": "^1.11",
+        "friendsofphp/php-cs-fixer": "^1.11,<1.11.5",
```

Comment:
I would rather update both 1.x line to use 2.1 and apply fixes, wdyt?
then, during merging 1.x to 2.x I will reapply the rules.